lua/telescope/_extensions/aerial.lua

```diff
@@ -128,6 +128,7 @@ local function aerial_picker(opts)
       default_selection_index = default_selection_index,
       sorter = conf.generic_sorter(opts),
       previewer = conf.qflist_previewer(opts),
+      push_cursor_on_edit = true,
```

Comment:
I'm really just copying telescope's built-in picker `lsp_document_symbols()`.

See the PR on the telescope repo here: https://github.com/nvim-telescope/telescope.nvim/pull/1234/files#diff-e706c69d9b98d70266879375b7a75301f9a48fde0fdb99fa85449972df202c16R155